undo_runner.py

```diff
@@ -16,6 +22,15 @@ def run(self, command_list):
               'was executed is "{}"'.format(last_line)
         prompt_input = raw_input('Do you want to undo that command? ([y]/N) ')
         if prompt_input == '' or prompt_input.lower() == 'y':
+
+            output = None
+
+            command_list = last_line.split()
+            if command_list[0] == 'add':
```

Comment:
We should abstract out the switcher to DRY it
https://github.com/juliexue/git-undo/blob/3cad2a6b0e6664629a1b628497bbe48eb2cb0076/parser.py#L32-L36

```python
    def get_command(self, command_list):
        switcher = {
                "commit": CommitCommand(command_list),
        }
        return switcher.get(command_list[0], UnknownCommand(command_list))
```